new/LIT6074TwelveFathers.xml

```diff
@@ -0,0 +1,83 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6074TwelveFathers" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዘአርትዑ፡ ነገረ፡ ፲ወ፪ አበው፡ ጠበብተ፡ መንፈሳውያን፡ ተጋብኡ፡ ኅቡረ፡ ወተናገሩ፡ በበይናቲሆሙ።</title>
+                <title xml:lang="en" xml:id="t2">The discourse of the twelve Fathers</title>
+                <title xml:lang="en" xml:id="t3">Sayings of twelve wandering monks</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl type="clavis">
+                     <bibl type="syriaca"><ptr target="bm:syriaca"></ptr><citedRange unit="item">1556</citedRange></bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p></p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="MonasticLiterature"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-07-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+
+            <div type="bibliography">
+              <listRelation>
+                <relation name="saws:formsPartOf" active="LIT6074TwelveFathers" passive="LIT4838CollectioM"></relation>
+                <relation name="saws:isAttributedToAuthor" active="LIT6074TwelveFathers" passive="PRS5585Isaiaht">
+                  <desc>The Syriac version of this text is attributed to <persName ref="PRS5585Isaiaht"></persName></desc>
```

Comment:
I think you could give a secondary reference to wright and the EAE for this record (and the next)